server/src/db/migrations/20220324210801_module_submissions_table.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema
+    .hasTable("ModuleSubmissions")
+    .then(function (exists) {
+      if (!exists) {
+        return knex.schema.createTable("ModuleSubmissions", function (t) {
+          t.increments("id").primary();
+          t.integer("moduleId").references("id").inTable("TrainingModule");
+          t.integer('makerId').references('id').inTable('Users');
```

Comment:
Can we rename these to `moduleID` and `makerID`?

Reply:
@MatthewGalan it's `FoobarId` everywhere though, maybe we should change those too

Reply:
Let's slowly migrate those over as well. We took a poll on the issue a while ago: https://discord.com/channels/879846554540855376/879846554540855379/949763875413303366